changelogs/unreleased/7008-sunjayBhatia-deprecation.md

```diff
@@ -0,0 +1,5 @@
+## useEndpointSlices feature flag removed
+
+As of v1.29.0, Contour has used the Kubernetes EndpointSlices API by default to determine the endpoints to configure Envoy with, instead of the Endpoints API.
```

Comment:
It could be helpful for the user if we'd like e.g. https://kubernetes.io/blog/2025/04/24/endpoints-deprecation/ for some background info here.